### PR TITLE
add diffusion velocity correction and timing function for dfHighSpeedFoam, correct alpha in rhoEEqn.H and clean the code

### PR DESCRIPTION
**Pull request by pkuLmq**

### CODE REVIEW

**Comment by zhixchen** [comment]
Can we still keep the "invisid" option?

**Comment by zhixchen** [comment]
This is to resolve issue #1

**Review by OpenFOAMFans — APPROVED** [review]

**Review by zhixchen — CHANGES_REQUESTED** [review]
The diffusion correction should be included in the "!invisid" condition.

**Review by zhixchen — APPROVED** [review]

File: applications/solvers/dfHighSpeedFoam/rhoEEqn.H [hunk 1]
```diff
@@ -30,11 +27,14 @@ if (!inviscid)
 {
     fvScalarMatrix eEqn
     (
-        fvm::ddt(rho, e) - fvc::ddt(rho, e)
+           fvm::ddt(rho, e) - fvc::ddt(rho, e)
         // alpha in deepflame is considered to calculate h by default (kappa/Cp), so multiply gamma to correct alpha
-      - fvc::laplacian(turbulence->alphaEff()*thermo.gamma(), e)
+         - fvc::laplacian(turbulence->alphaEff()*thermo.gamma(), e)
```
[the remark this answers is not in the document]
Comment (zhixchen) [reply applications/solvers/dfHighSpeedFoam/rhoEEqn.H:32]: FVM is compatiable with the current energy definition. This is to be further investigated in a future version.

**Pull request merged**